Application/EdFi.Ods.AdminApi/Program.cs

```diff
@@ -17,7 +17,21 @@
 builder.Services.AddSingleton<IIpPolicyStore, MemoryCacheIpPolicyStore>();
 builder.Services.AddSingleton<IRateLimitCounterStore, MemoryCacheRateLimitCounterStore>();
 builder.Services.AddSingleton<IRateLimitConfiguration, RateLimitConfiguration>();
-builder.Services.AddInMemoryRateLimiting();
+builder.Services.AddInMemoryRateLimiting();
+var enableCors = builder.Configuration.GetValue<bool>("CorsSettings:EnableCors");
+string allowAllCorsPolicyName = "allowAllCorsPolicyName";
+if (enableCors)
+{
+    builder.Services.AddCors(options =>
+    {
+        options.AddPolicy(allowAllCorsPolicyName, policy =>
+        {
+            policy.AllowAnyOrigin()

```

Comment:
This is ok for development environment, but we have to consider adding the policy for qa/production environment, so at least include the origins that we are allowing to use it

`policy.WithOrigins([list of url values from appsettings])`

Reply:
Done